common/CommunityToolkit.Labs.Shared/Helpers/PackageProjectUrlAttribute.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace CommunityToolkit.Attributes;
```

Comment:
Namespace doesn't match folder structure. We're likely to have more attributes like this in the near future, let's move this to a dedicated `Attributes` folder.

Reply:
Yeah, was trying to avoid making a folder for one file... 😋